share/wake/lib/system/job.wake

```diff
@@ -320,6 +320,20 @@ export def runJob p = match p
         def _ = wait (\_ badfinish job error) (badlaunch job error)
         job
 
+
+# Get result for job with the list of output files/dirs.
+export def getJobResultOutputs (job: Job) : Result (List Path) Error =
+  map getPathResult job.getJobOutputs
+  | findFail
+
+
+# Get result for job with exactly one output Path.
+export def getJobResultOutput (job: Job) : Result Path Error =
+  match job.getJobOutputs
```

Comment:
Somehow I overlooked this PR. Unfortunately, this line is no good, because a failing job has a single BadPath output.

Reply:
Is the edit I just pushed what you're getting at, or did I misunderstand what you're wanting?

Reply:
Yes